meteor/client/lib/viewPort.ts

```diff
@@ -113,14 +114,60 @@ export function scrollToSegment(
 	noAnimation?: boolean,
 	secondStage?: boolean
 ): Promise<boolean> {
-	let elementToScrollTo: HTMLElement | null = isProtectedString(elementToScrollToOrSegmentId)
-		? document.querySelector('#' + SEGMENT_TIMELINE_ELEMENT_ID + elementToScrollToOrSegmentId)
-		: elementToScrollToOrSegmentId
+	const getElementToScrollTo = (showHistory: boolean): HTMLElement | null => {
+		if (isProtectedString(elementToScrollToOrSegmentId)) {
+			console.log(elementToScrollToOrSegmentId)
+			let targetElement = document.querySelector<HTMLElement>(
+				`#${SEGMENT_TIMELINE_ELEMENT_ID}${elementToScrollToOrSegmentId}`
+			)
+
+			console.log(targetElement)
+			console.log(Settings.followOnAirSegmentsHistory)
+			console.log(targetElement?.parentElement?.parentElement?.previousElementSibling)
```

Comment:
I suppose these should be removed?

Reply:
Ah! Yes, I have removed them here https://github.com/olzzon/tv-automation-server-core/commit/16eff9414cc2d34d93707c8fc0d6eebbb6ca57cd